server/index.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express')
+const bodyParse = require('body-parser')
+const bcrypt = require('bcrypt')
+const mongoose = require('mongoose')
+const user = require('./models/user')
+
+mongoose.Promise = Promise
+
+const promise = mongoose.connect('mongodb://<username>:<password>@ds127936.mlab.com:27936/bears-08', {
```

Comment:
How about using an env variable for the entire host instead of the hardcoded? 
```js
const promise = mongoose.connect(process.env.DB_HOST, {
    ...
}) 
```
It will be easier to set up local dev, public, and ci later by swapping env variables.